### DISCUSSION TITLE
Data behind Confusion Matrix

### DISCUSSION THREAD

**Discussion by cool112624** [ultralytics/yolov5#9635] [Q&A]
Hi, I want reduce False Negative on confusion matrix of a model that I train

So I want to know which data is causing the false negative prediction so can I anywhere export the data that represent in each square of confusion matrix

like how can I get the data that are in this red square?

Thank you in advance 
![confusion_matrix](https://user-images.githubusercontent.com/51593747/192916317-2f3566e8-4c40-47b2-a089-31b6b969e3b7.png)

**Comment by cool112624** [comment]
@glenn-jocher Hi Glenn do you have any idea that can help me out ?

**Reply by glenn-jocher** [reply]
@cool112624 for confusion matrix code see
https://github.com/ultralytics/yolov5/blob/799e3d0cc92a9f431d97931641e7d0b46720699a/utils/metrics.py#L126-L223
